src/lib/y2network/hwinfo.rb

```diff
@@ -16,14 +16,36 @@
 #
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
-require_relative "../test_helper"
-require "y2network/config_writer"
+module Y2Network
+  # FIXME: should read hwinfo for a network device and store only necessary info
+  class Hwinfo
```

Comment:
so goal is to have HWInfo per interface?

Reply:
We already had a reader in storage-ng: https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2storage/hwinfo_reader.rb

Reply:
@jreidinger:
purpose is to store only what is needed. Hw info output is large and we need only couple of values. Also some values are used e.g. once other multiple times. It can turn out that using a singleton as pointed by @imobachgs. We will see ... but we definitely need some api for accessing it otherwise than touching the hash map - that's the point of this class for now.

Reply:
ok, so goal of class is to have network friendly layer on top of hwinfo hash (so no direct access to hwinfo should be done). Then please document it in class description.